lightningd/plugin_control.c

```diff
@@ -267,30 +273,34 @@ static struct command_result *json_plugin_control(struct command *cmd,
 		return plugin_dynamic_stop(cmd, plugin_name);
 	} else if (streq(subcmd, "start")) {
 		const char *plugin_path;
+		unsigned int *timeout_seconds;
 
 		if (!param(cmd, buffer, params,
 			   p_req("subcommand", param_ignore, cmd),
 			   p_req("plugin", param_string, &plugin_path),
+			   p_opt_def("timeout", param_number, &timeout_seconds, 3),
```

Comment:
I think this is one of those cases where adding a parameter doesn't help.  3 seconds is probably too short, but 30 seconds is probably about right.  If it's not, I think we need to fix it, not the users?

Reply:
Is letting the possibility for the command to be pending for 30s ok ?

Reply:
In this case, the `sleep` needed for the timeout test in `test_plugin_slowinit` seems overkill